src/psyclone/domain/lfric/kern_call_arg_list.py

```diff
@@ -993,7 +993,7 @@ def cell_ref_name(self, var_accesses=None):
         if self._kern.is_coloured():
             colour_sym = self._symtab.find_or_create_integer_symbol(
                 "colour", tag="colours_loop_idx")
-            array_ref = self.get_array_reference("cmap",
+            array_ref = self.get_array_reference(self._kern.colourmap,
```

Comment:
Is there not a test file associated with `kern_call_arg_list` that should be updated due to this change?

Reply:
Actually, this was just to protect against potential name clashes so nothing changed. I've now altered a couple of the test files to provoke name clashes to demonstrate that everything is OK.